components/wb-urlmapping/selectinput-en.html

```diff
@@ -0,0 +1,224 @@
+---
+{
+	"title": "URL Mapping - Select input",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Select inputs by default based on url query string.",
+	"tag": "urlmapping",
+	"parentdir": "urlmapping",
+	"altLangPage": "selectinput-fr.html",
+	"dateModified": "2023-10-24"
+}
+---
+<ul class="list-inline">
+	<li><a class="btn btn-primary" href="urlmapping-doc-en.html">Documentation</a></li>
+	<li><a class="btn btn-primary" href="https://github.com/wet-boew/GCWeb/issues/new?title=UrlMapping+tblFilter">Questions or comments?</a></li>
+</ul>
+
+<p>Select inputs by default based on url query string.</p>
+
+<div class="wb-prettify all-pre"></div>
+
+<ul>
+	<li><a href="?">Empty <code>?</code></a></li>
+	<li><a href="?f=b">Bananas <code>?f=b"</code></a></li>
+	<li><a href="?f=g">Grapes <code>?f=g"</code></a></li>
+</ul>
+
+<section>
+	<h2>Example with <code>source</code></h2>
+	<div data-wb-urlmapping='{
+		"f=b": {
+			"action": "selectInput",
+			"source": "#fruitsOptions",
+			"value": "Bananas"
+		}
+	}'></div>
+
+	<form id="fruitsOptions">
+		<label for="fruitsSelect">Choose a fruit</label>
```

Comment:
I suggest to mention the context in order to differentiate the other fieldset named the same

Something like:
```suggestion
		<label for="fruitsSelect">(<code>&lt;select></code>) Choose a fruit</label>
```